doors_dashboards/dashboards/dashboard.py

```diff
@@ -113,6 +116,18 @@ def create_dashboard(config: Dict) -> html.Div:
         ]
     )
 
+    @callback(
+        Output("none", "children",allow_duplicate=True),
+        [Input(f"{dashboard_id}-interval", "n_intervals")],
+        prevent_initial_call=True
+    )
+    def get_new_data(interval):
+        collections = feature_handler.get_collections()
+        for collection in collections:
+            feature_handler.delete_df(collection)
+        #for collection in collections:
+        #feature_handler.get_df(collection)
```

Comment:
```suggestion
        for collection in collections:
            feature_handler.delete_df(collection)
            feature_handler.get_df(collection)
```

I reconsidered. I like this option the best.